root/tag/utility/buildTagListSection.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import ENTITIES from '../../../entities';
+import DescriptiveLink
+  from '../../static/scripts/common/components/DescriptiveLink';
+import expand2text from '../../static/scripts/common/i18n/expand2text';
+
+
+type Props = {
```

Comment:
It's not quite the same - this has optional booleans (because they don't exist for TagIndex) but UserTag has mandatory ones. Might not be worth the distinction though, I dunno.